src/encoded/static/components/file.js

```diff
@@ -579,6 +579,18 @@ class SequenceFileInfo extends React.Component {
                             </div>
                         : null}
 
+                        {file.index_of ?
+                            <div data-test="outputtype">
+                                <dt>Index of</dt>
+                                <dd>
+                                    {file.index_of.map((f) => {
```

Comment:
react refuse to work with using map and join. After searching online, some said use map and reduce. See- https://stackoverflow.com/a/40788571/178550

Reply:
Wow! I’ve never thought of that. Kind of a mind bender, and that it keeps nesting arrays on each iteration is kind of a shame, though as that post says, React 16 should have no issue with that. I think I would probably use this myself going forward though.

Reply:
@forresttanaka Addressed, based on- https://stackoverflow.com/a/47459785/178550